modules/ROOT/pages/mule-upgrade-tool.adoc

```diff
@@ -425,6 +427,26 @@ The Mule upgrade tool requires both the current Mule instance and the new Mule i
 
 Perform a manual upgrade to ensure that the current and new Mule instances are in version 4.1.5 or later before attempting to upgrade using the Mule upgrade tool.
 
+[[upgrade_clustered_mule_instances]]
+== Upgrading clustered Mule instances
+
+Currently, the upgrade tool does not automatically support upgrading Mule clustered instances, and because of that you can find here a description of what to do to upgrade it using the Mule upgrade tool.
+
+=== Before You Begin
+
+To ensure less downtime, first copy the new instance of the Mule runtime and the Mule upgrade tool to each instance that will be upgraded. That way you can have everything you need to do the upgrade.
+Follow the same procedures as are described in the <<before_you_begin>> section.
+
+=== Upgrade or Update Mule
+
+The procedure you need to follow is the same as in the stand-alone instance upgrade.
+Follow the same procedure described above, starting from the server node, and continuing with every other node till each of those are upgraded to the same version.
+This way your system will be properly updated and working as a cluster with the proper new version.
+
+=== Rollback Mule
+
+The same thing that happens with the upgrade happen with the rollback procedure, every node must be rolled back and ends up having the same version. Because of that, you will need to roll back every node in the same order as the upgrade, first the server node, and then each individual node that is part of the cluster, making sure that, at the end, every node has the same version.
```

Comment:
```suggestion
Before starting an upgrade or update process, the Mule upgrade tool generates a backup of your existing clustered Mule instances, which enables you to restore the upgraded or updated clustered Mule instances to their previous state. The backup is stored in the `$MULE_HOME/.upgrade-tool-backup` folder.
To perform a rollback:
. Follow the procedures described in the above <<roll_back_an_upgrade-or-update>> section. Every node must be rolled back in the same order as they were upgraded or updated starting from the server node and continuing with the other nodes that are part of the cluster. Once you finish, make sure that every node has the same version.
```
1. Added some info as per the <<roll_back_an_upgrade-or-update>> tool description considering that I want to modify the navigation tree as a next step to improve customer experience. Please check if it's OK. 
2. What do you mean by "same version"? Are we referring to the Original version?

Reply:
Done